src/main/java/org/mcmonkey/denizen2core/commands/queuecommands/RunCommand.java

```diff
@@ -26,11 +26,15 @@
     // @Maximum 2
     // @Tag <def[run_queue]> (IntegerTag) returns the qID of the ran queue.
     // @Description
-    // Runs a script as a new queue.
+    // Runs a script as a new queue. Can pass definitions.
     // TODO: Explain more!
     // @Example
     // # This example runs the script "test".
     // - run test
+    // @Example
+    // # This example runs the script "myTask" with definitions "banana"
+    // and "potato" of values "2" and "5".
```

Comment:
`#` symbol, because you're writing script comments here.

Also, careful with the English. `with values` not `of values`.